app/src/main/java/io/homeassistant/companion/android/vehicle/EntityGridVehicleScreen.kt

```diff
@@ -160,8 +163,19 @@ class EntityGridVehicleScreen(
                 gridItem
                     .setOnClickListener {
                         Log.i(TAG, "${entity.entityId} clicked")
-                        lifecycleScope.launch {
-                            entity.onPressed(integrationRepository)
+                        if (entity.domain in MainVehicleScreen.MAP_DOMAINS) {
+                            val attrs = entity.attributes as Map<*, *>
+                            val lat = attrs["latitude"] as Double
```

Comment:
This attribute may no longer exist (removed after adding the favorite), add a check to handle null.

Reply:
thanks for calling that out, made the check more consistent  here too